Library/Homebrew/sandbox.rb

```diff
@@ -141,8 +141,15 @@ def path_filter(path, type)
   class SandboxProfile
     SEATBELT_ERB = <<-EOS.undent
       (version 1)
+      (deny network-inbound)
+      (deny network-outbound)
       (debug deny) ; log all denied operations to /var/log/system.log
       <%= rules.join("\n") %>
+      (allow network-outbound
+        (subpath "/tmp")
+        (subpath "/private")
+        (subpath "#{HOMEBREW_TEMP}")
```

Comment:
> Because I have not idea how to implement this without breaking it. It's much harder to create a whitelist on read operation than write.

Yeh. It's a shame that you can tell read operations to e.g. just not tell there's a file there rather than erroring out.



> A good suggestion would be implementing methods such as allow_read_*, allow_network_* similar to allow_write_* instead of hard coding the rule.

Agreed. I guess we could whitelist the upstream formula URL(s)?